### DISCUSSION TITLE
[BE] DB 테이블에 null값이 들어가도 괜찮을까?

### DISCUSSION THREAD

**Discussion by Seongwon97** [woowacourse-teams/2022-momo#11] [General]
모임 생성을 할 때, 장소와 시간은 선택 사항으로 받고 있다.
입력값이 들어왔을 때는 문제가 없지만 해당 값을 입력 받지 않았을 때는 서비스단으로부터 DB까지 Null값이 들어가게 된다.

해당 값의 경우 Null값을 그대로 넣어두는게 좋을까? 아니면 임의의 값으로 변경을 해서 넣어주는 것이 좋을까 생각을 해봐야겠다.

**Comment by kyukong** [comment]
테이블에 null 이 들어가더라도 불러올 때 그에 대한 방어 코드만 제대로 작성한다면 null 을 입력해도 로직상의 문제는 없을 것 같습니다. 다만, 테스트를 제대로 작성하지 않았다면 데이터가 저장되고 한참 후에야 그 사실이 발견되어 언제, 어디에서 null 이 발생한건지 파악하기 어려울 수도 있을 것 같습니다. 처음부터 null 입력을 막아서 방지하는 것은 어떨까요?

**Reply by sinb57** [reply]
저 또한 DB에서 null이 저장되는 행위를 막는 것에 동의합니다!!
